src/components/slider/slider.st.css

```diff
@@ -0,0 +1,140 @@
+:import {
+    -sb-from: "../../style/default-theme/variables.st.css";
+    -sb-named: color_Border,
+    color_Slider_Track,
+    color_Slider_Track_Hover,
+    color_Slider_Progress,
+    color_Slider_Progress_Hover,
+    color_Slider_Handle,
+    color_Slider_HandleBorder,
+    color_Slider_Handle_Hover,
+    color_Slider_Handle_Active;
+}
+
+:vars {
+    trackHeight: 4px;
+    handleMargin: -6px;
+    handleWidth: 16px;
+    handleHeight: 16px;
+    handleRadius: 8px;
+    handleBorder: 1px solid value(color_Slider_HandleBorder);
+    handleBorderHover: 1px solid value(color_Slider_Handle_Active);;
+}
+
+.root {
+    -st-states: active, disabled, error;
+    width: 100%;
+    position: relative;
+    display: inline-block;
+}
+
+.slider {
+    width: 100%;
```

Comment:
if this is considering 'take all width', then better is to use `display: block`
but I think slider should be `inline-block` element with some default `min-width` value and then good api to make it block/flex-item so it will take all given container width